lib/exit.js

```diff
@@ -103,9 +103,43 @@ export default class Exit {
 
     return (signer
       ? signer.signTx(spendTx)
-      : spendTx.signWeb3(rootChain)).then(signedTx => plasmaChain.eth.sendSignedTransaction(
-      signedTx.hex()
-    )).then(transferTx => {
+      : spendTx.signWeb3(rootChain)).then(signedTx => {
+        return new Promise((resolve, reject) => {
+          plasmaChain.currentProvider.send({
```

Comment:
we are using this combo in multiple places. Maybe let's extract it in a separate helper?

Reply:
Added `sendSignedTransaction` to helpers.